src/csharp/Microsoft.Spark.Worker/Processor/PayloadProcessor.cs

```diff
@@ -87,6 +74,15 @@ internal Payload Process(Stream stream)
             }
 
             payload.IncludeItems = ReadIncludeItems(stream);
+
+#if NETCOREAPP
```

Comment:
so basically, .NET framework will not work right? If so, let's remember to put this in the release note.

Reply:
I suppose we don't really need this `#if`.  It should compile fine with either framework.  The only issue I foresee is the `dotnet-interactive` session is only pulling in dotnet and netstandard nugets. The `AssemblyProbingPaths` would only have paths to these two frameworks, and if a Worker required a `net461` specific nugets, then it wouldn't be available.